### PR TITLE
Added RSpec and fixed code style

### PR DESCRIPTION
**Pull request by Minpeople**

### CODE REVIEW

File: app/controllers/cinema_halls_controller.rb [hunk 1]
```diff
@@ -1,4 +1,48 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+  def index
```
Comment (kodowicz) [inline app/controllers/cinema_halls_controller.rb:4]: You haven't closed another PR and this changes shouldn't be included in this PR :) Here are some changes that are not connected with rspec branch

File: test/channels/application_cable/connection_test.rb [hunk 2]
```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
-require "test_helper"
```
Comment (kodowicz) [inline test/channels/application_cable/connection_test.rb:3]: If you are using `rspec` you do not need `test/` so you can remove them. All your tests will be located in `spec/` directory

**Comment by wmaciejak** [comment]
@Minpeople why this PR is closed? :)
